server/src/static/js/content-helper.js

```diff
@@ -33,24 +33,40 @@ window.addEventListener(
 let highlightElement;
 
 function displayClip(clip) {
-  if (clip.text) {
-    return;
-  }
-  let loc = clip.image.location;
+  let topLeft = null;
+  let bottomRight = null;
+  let loc = null;
   let pos = {
-    top: loc.top,
-    bottom: loc.bottom,
-    left: loc.left,
-    right: loc.right
+    top: 0,
+    bottom: 0,
+    left: 0,
+    right: 0
   };
-  let topLeft = findElement(loc.topLeftElement);
+
+  if (clip.text) {
+    topLeft = findElement("#" + clip.text.location.selectionStart);
```

Comment:
Did I use inconsistent selectors for text and images?  Yuck.  I guess maybe for now we should open a ticket to straighten that out.